Bucketeer2/Sources/Internal/DI/Component.swift

```diff
@@ -9,29 +9,17 @@ protocol Component: AnyObject {
 
 final class ComponentImpl: Component {
     let dataModule: DataModule
+    let evaluationInteractor: EvaluationInteractor
+    let eventInteractor: EventInteractor
 
     init(dataModule: DataModule) {
         self.dataModule = dataModule
-    }
-
-    var config: BKTConfig {
-        dataModule.config
-    }
-
-    var userHolder: UserHolder {
-        dataModule.userHolder
-    }
-
-    var evaluationInteractor: EvaluationInteractor {
```

Comment:
The computed property pattern has a bug that the cache data was not shared between instances.